src/common/transformations/include/transformations/common_optimizations/prelu_fusion.hpp

```diff
@@ -103,13 +103,23 @@ class ngraph::pass::PReluFusionMultiplySub : public ngraph::pass::MatcherPass {
  * @ingroup ie_transformation_common_api
  * @brief PReluFusion transformation replaces various sub-graphs with a PRelu op.
  */
-class ngraph::pass::PReluFusion : public ngraph::pass::GraphRewrite {
+class ov::pass::PReluFusion : public ov::pass::GraphRewrite {
 public:
     NGRAPH_RTTI_DECLARATION;
```

Comment:
please replace it with OPENVINO_RTTI("PReluFusion", "0"); and delete NGRAPH_RTTI_DEFINITION(...) from cpp files
the same for other transformations